.circleci/config.yml

```diff
@@ -869,18 +919,19 @@ jobs:
                 echo "User '${CIRCLE_USERNAME}' is allowed to force pack uploads."
             fi
 
-      - run:
-          name: Upload Packs To Marketplace Storage
-          command: |
-            EXTRACT_FOLDER=$(mktemp -d)
-            PACK_ARTIFACTS=$CIRCLE_ARTIFACTS/content_packs.zip
-            PACKS_DEPENDENCIES=$CIRCLE_ARTIFACTS/packs_dependencies.json
-            ID_SET=$CIRCLE_ARTIFACTS/id_set.json
-            GCS_PATH=$(mktemp)
-            echo $GCS_MARKET_KEY > $GCS_PATH
-            python3 ./Tests/Marketplace/upload_packs.py -a $PACK_ARTIFACTS -e $EXTRACT_FOLDER -b 'marketplace-dist' -d $PACKS_DEPENDENCIES -i $ID_SET -s $GCS_PATH -n $CIRCLE_BUILD_NUM -k $PACK_SIGNING_KEY -pb 'marketplace-dist-private' -p "${PACKS_TO_UPLOAD}" -o
-            rm $GCS_PATH
-          when: on_success
+      - *force_upload_packs_to_marketplace_storage
+      - *destroy_instances
```

Comment:
why did you add that for the force upload ?

Reply:
moved the step to the `references` section, just to make things nicer